software-containers/arcade-casa/version-3.4-4.2/startup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#echo "sourcing admit_start.sh"
+#source /opt/admit/admit_start.sh
```

Comment:
Why is this commented out?

Reply:
Not needed. Removed. There is no 'ADMIT' for these CASA versions.